utils/clients/ratios/client.go

```diff
@@ -40,7 +42,24 @@ func NewWithContext(ctx context.Context) (Client, error) {
 	if err != nil {
 		return nil, err
 	}
-	return NewClientWithPrometheus(&HTTPClient{client}, "ratios_context_client"), err
+
+	// get default timeout and purge from context
+	expires, err := appctx.GetDurationFromContext(ctx, appctx.RatiosCacheExpiryDurationCTXKey)
+	if err != nil {
+		expires = 5 * time.Second
+	}
+
+	// get default purge and purge from context
+	purge, err := appctx.GetDurationFromContext(ctx, appctx.RatiosCachePurgeDurationCTXKey)
+	if err != nil {
+		purge = 1 * time.Minute
```

Comment:
I know there's a `RATIOS_CACHE_PURGE` environment variable, might be a minor bug we're not using it?

Reply:
We are using it, it is passed through the context.  Cobra commands insert from the BindEnv value into the context under this key.